code/modules/religion/burdened/psyker.dm

```diff
@@ -197,6 +218,13 @@
 /obj/item/gun/ballistic/revolver/chaplain/attack_self(mob/living/user)
 	pray_refill(user)
 
+/obj/item/gun/ballistic/revolver/chaplain/attackby(obj/item/possibly_ammo, mob/user, params)
+	if (isammocasing(possibly_ammo) || istype(possibly_ammo, /obj/item/ammo_box))
+		to_chat(user, span_warning("To reload manually? Sacrilege."))
```

Comment:
```suggestion
		user.balloon_alert(user, "no manual reloads!")
```

i know it lacks flavor and sovl but this is better UX
i know it's less flavorful and lacks sovl but this is better ux

Reply:
That's fine.